University.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Week4_HomeWork
+{
+    public class University
+    {
+        public string _name;
+        public List<Student> students = new List<Student>();

```

Comment:
Try to use the same naming convention everywhere in the project. Some fields have names starting with _, others without.